src/index.html

```diff
@@ -6,13 +6,69 @@
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
     <title>Product cards</title>
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <link
+      rel="stylesheet"
+      href="./styles/variables.scss"
+    />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="product-card"
+      data-qa="card"
+    >
+      <img
+        class="product-card__image"
+        src="./images/imac.jpeg"
+        alt="APPLE A1419 iMac"
+      />
+      <div class="product-card__content">
+        <h2 class="product-card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <p class="product-card__code">Product code: 195434</p>
+        <div
+          class="product-card__rating"
+          data-qa="hover"
+        >
+          <div class="stars">
+            <div class="stars__star stars__star--active"></div>
+            <div class="stars__star stars__star--active"></div>
+            <div class="stars__star stars__star--active"></div>
+            <div class="stars__star stars__star--active"></div>
+            <div class="stars__star"></div>
+          </div>
```

Comment:
```suggestion
          <div class="stars stars--4">
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
          </div>
```
Consider using the layout and styles from the previous task here